tests/vm_test.go

```diff
@@ -2073,53 +2074,3 @@ func createRunningVM(virtClient kubecli.KubevirtClient, template *v1.VirtualMach
 	Expect(err).ToNot(HaveOccurred())
 	return vm
 }
-
-func startVM(virtClient kubecli.KubevirtClient, vm *v1.VirtualMachine) *v1.VirtualMachine {
```

Comment:
Please post chages of "private" implementations (i.e. not from utils) in a separate commit so we can examine that no logic changed when doing so.

I think there is some mix with the use of `Running` and `RunStrategy` and there me more other points.
You also have the opportunity to replace these `Update` operation with a `Patch`. I think such a change was already done by a code-quality PR not long ago.

Reply:
ack